### DISCUSSION TITLE
Restarting server via Rooster

### DISCUSSION THREAD

**Discussion by milivojm** [CntoDev/level-up#114] [Ideas]
A Rooster user logs in Rooster and if he has a permission to restart the server, there should be a page to do it. Nothing fancy, just restarting with latest configuration is good enough for starters.

It seems doable if `Main` can be somehow connected to `Tools` server or other way around.

**Comment by milivojm** [comment]
Possible solutions:
- Connect **Luncher** to RabbitMq and on `RestartServer` command, restart Arma server.
- Build a REST API and deploy it on `Main`. Call API from **Rooster**. This one seems kind of meh because deploying Web API on `Main` is blah.

**Comment by enrico-ghidoni** [comment]
Cool feature! I would definitely go with connecting Luncher to RabbitMQ, seems like the easiest solution

**Comment by JamesTheClarke** [comment]
Love that idea! Would also be much easier on admins to share that access without needed actual vpn access to Seneram's infrastructure.

**Reply by JamesTheClarke** [reply]
Just to clarify, this is not needed for an initial release. But it is a very nice feature for a future update.

v1 really should focus mainly on getting two things right:

1.) handling applications with the same features currently used in enjin and old roster
2.) handling of member management with the same features as the old roster

Automated permission management (including server restarts etc.) is more of a long term goal in my view.
